public/js/actions/breakpoints.js

```diff
@@ -24,8 +32,16 @@ function _getOrCreateBreakpoint(state, location, condition) {
   return getBreakpoint(state, location) || fromJS({ location, condition });
 }
 
-function addBreakpoint(location, { condition, getTextForLine } = {}) {
-  return ({ dispatch, getState, client }) => {
+type AddBreakpointOpts = {
+  condition?: string,
+  getTextForLine?: (line: number) => string
+};
+
+function addBreakpoint(
+  location: Location,
+  { condition, getTextForLine } : AddBreakpointOpts = {}
```

Comment:
Maybe it's because I'm new to this, but `AddBreakpointOpts` seems unhelpful and adds a lot of clutter